### ISSUE TITLE
Running wsimport with JDK 16 throws InaccessibleObjectException

### ISSUE THREAD

**Issue by Thihup** [OpenLiberty/open-liberty#17350]
**Describe the bug**
Running wsimport with JDK 16 throws InaccessibleObjectException

**Steps to Reproduce**
```
$ ./bin/xmlWS/wsimport
[...]
java.lang.ExceptionInInitializerError
        at com.sun.tools.ws.wscompile.WsimportTool.run(WsimportTool.java:244)
        at com.sun.tools.ws.wscompile.WsimportTool.run(WsimportTool.java:172)
        at io.openliberty.xmlWS30.tools.WsImport.main(WsImport.java:77)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:567)
        at com.ibm.ws.kernel.boot.cmdline.UtilityMain.internal_main(UtilityMain.java:173)
        at com.ibm.ws.kernel.boot.cmdline.UtilityMain.main(UtilityMain.java:53)
        at com.ibm.ws.kernel.boot.cmdline.Main.main(Main.java:52)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private static volatile java.net.Authenticator java.net.Authenticator.theAuthenticator accessible: module java.base does not "opens java.net" to unnamed module @3ac42916
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:177)
        at java.base/java.lang.reflect.Field.setAccessible(Field.java:171)
        at com.sun.istack.tools.Utils.lambda$getCurrentAuthenticator$0(Utils.java:38)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:312)
        at com.sun.istack.tools.Utils.getCurrentAuthenticator(Utils.java:37)
        at com.sun.istack.tools.DefaultAuthenticator.getCurrentAuthenticator(DefaultAuthenticator.java:208)
        at com.sun.istack.tools.DefaultAuthenticator.<clinit>(DefaultAuthenticator.java:43)
        ... 10 more
```

**Expected behavior**
Exception not being thrown

**Diagnostic information:**
 - OpenLiberty Version: 21.0.0.6-beta
 - Java Version:
openjdk version "16" 2021-03-16
OpenJDK Runtime Environment (build 16+36-2231)
OpenJDK 64-Bit Server VM (build 16+36-2231, mixed mode, sharing)

**Additional context**
From what I've searched, updating the IStack library would fix this issue, because they are using a multi-release jar:
https://github.com/eclipse-ee4j/jaxb-istack-commons/blob/master/istack-commons/tools/src/main/java-mr/9/com/sun/istack/tools/Utils.java
However, an easier fix would be to include `--add-opens java.base/java.net=ALL-UNNAMED` to the `wsimport` script.
I could send a PR if appropriate for the second case.

**Comment by gjwatts** [comment]
Good catch @Thihup.  We will take a look into this, thanks for letting us know!

**Comment by neuwerk** [comment]
@Thihup really appreciate the heads up! It looks like I should update the version of istack we are using, we'd just updated a whole set of others but missed this one. I'll hopefully have an update for you shortly.

**Comment by neuwerk** [comment]
I've merged #17370 which bring us up to date with the current version of IStack. I've not tested this to make sure it resolves the issue in xmlWS-3.0's wsimport, as the update needed to be done regardless but I will update this issue once I've done so.

**Comment by gjwatts** [comment]
@Thihup just following up on this.  I was able to successfully run the `wsimport` command under both the `xmlWS` and `jaxws` directories on Java 16 and 17.  I tested with our current 21.0.0.11-beta version.
Closing.
